BenadrylAutomation/src/main/java/com/jnj/actions/Action.java

```diff
@@ -264,7 +264,7 @@ public static boolean selectByVisibleText(String visibletext, WebElement ele, St
 			if (flag) {
 				BaseClass.extentInfoLog(log, "");
 			} else {
-				BaseClass.extentInfoLog("Option not selected by VisibleText", "");
+				BaseClass.extentFailLog("Option not selected by VisibleText : ", visibletext);

```

Comment:
@goutamnaik007 if this is FailLog then pls update line 265 as PassLog. Also, put message in "".

Reply:
We dont need to upadte the info log to pass log, if the text is found with visible text , it will select the value and just log the value selected. But if it does'nt if mark it as failure
@vaibhavNagvekarSJI

Reply:
ok fixed